src/applications/caregivers/components/ToggleButton/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default function ToggleButton({
```

Comment:
This actually looks like it is so re-usable, a better name might just be ToggleComponent.

Reply:
It is being removed